ReadyPlayerOne/Models/Player.cs

```diff
@@ -34,11 +34,11 @@ public class Player
         // PlayerImage is its own class so I can pull the images easier.
         [ForeignKey("PlayerImage")]
         public string PlayerImageID { get; set; } = string.Empty;
-        public PlayerImage PlayerImage { get; set; } = null!;
+        public PlayerImage? PlayerImage { get; set; }
```

Comment:
Set the variable to `?` and remove setting the value to null. It breaks the links most time.

Reply:
That is good to know, thank you!